### PR TITLE
New Developer Setup and Build System

### PR DESCRIPTION
**Pull request by cuahsimarko**
### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. [Enter positive test case here]

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,7 +14,6 @@
 /celery
 /config/hydroshare-config.sh
 /config/hydroshare-config.yaml
-/hydroshare/local_settings.py
 /hydroshare/static
```
Comment (cuahsimarko) [inline .gitignore]: Just to triple check: keep an eye on this and ensure it doesn't negatively impact the build system in Jenkins

File: Dockerfile-data-irods [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM mjstealey/docker-irods-icat:4.1.8
+
```
Comment (cuahsimarko) [inline Dockerfile-data-irods]: Phuong and I looked into updating the image here but it needs to wait

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,8 @@
+## iRODS
+rods@data.local.org.env
+rods@users.local.org.env
+
+## Custom
 *.pyc
```
Comment (cuahsimarko) [inline .gitignore]: this logic will eventually move to irods config files or dockerfiles

**Comment by cuahsimarko** [comment]
These files are all inactive and are simply being staged for cutover, after core developers have adopted the new dev setup and the Jenkins build is complete (in progress 1/3/2019)

File: Dockerfile-data-irods [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+ADD ./irods/delete_user.sh /home/hsuserproxy
+ADD ./irods/delete_user.sh /home/hsuserproxy
+
+RUN useradd -m -p hsuserproxy -s /bin/bash hsuserproxy
+
+RUN chmod -R 777 /home
```
Comment (hyi) [inline Dockerfile-data-irods]: @cuahsimarko I believe line 3-6 are not needed here.

Comment (cuahsimarko) [reply Dockerfile-data-irods]: Ok I added to my notes for subsequent optimizations

**Pull request merged**